worker/include/RTC/RTCP/FeedbackRtpTransport.hpp

```diff
@@ -213,6 +213,7 @@ namespace RTC
 
 		public:
 			AddPacketResult AddPacket(uint16_t sequenceNumber, uint64_t timestamp, size_t maxRtcpPacketLen);
+			void SetBase(uint16_t sequenceNumber, uint64_t timestamp);
```

Comment:
`SetBase()` should be first since `AddPacketResult()` should not be called if `SetBase()` was not called.

```suggestion
			void SetBase(uint16_t sequenceNumber, uint64_t timestamp);
			AddPacketResult AddPacket(uint16_t sequenceNumber, uint64_t timestamp, size_t maxRtcpPacketLen);
```